ee/billing/quota_limiting.py

```diff
@@ -89,6 +92,14 @@ def org_quota_limited_until(organization: Organization, resource: QuotaResource)
     if is_quota_limited and organization.never_drop_data:
         return None
 
+    if posthoganalytics.feature_enabled(QUOTA_LIMIT_DATA_RETENTION_FLAG):
```

Comment:
This is probably sending more events than we would like (i.e. one when quota limiting starts, one when quota limiting ends).

Reply:
Discussed with @raquelmsmith and we decided that this is fine. Since the flag is manually triggered, we may see a short spike in the number of 'quota limiting suspended' events when we enable it. This would also serve as confirmation that the logic here is working.